apps/tasks/.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
```

Comment:
Is this another instance of a file that could inherit from the top-level formatter or do we want to have one per-app?

Reply:
Top level should be fine. Nice catch.